content/posts/installing-arch-linux-on-uefi-with-full-disk-encryption.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Installing Arch Linux on UEFI with Full Disk Encryption
+author: Lena Fuhrimann
+comments: true
+date: 2021-03-12
+---
+
+This is a step by step guide to installing Arch Linux on UEFI with full disk encryption. It deliberately contains no unnecessary words or bling.
+
+## Download ISO
+
+1. Download the latest ISO from the [Arch Linux website](https://archlinux.org/download/)
+
+## Create Bootable USB Stick
+
+You can skip this step if you just want to run Arch Linux in a VM. In that case, just run the ISO from your favorite VM management tool like [QEMU](https://www.qemu.org/).
+
+1. Insert an USB stick into your computer
+1. Run `lsblk` to find the correct disk
+1. Run `sudo unmount /dev/sdx` or whatever the USB stick is
+1. Run `sudo dd bs=4M if=path/to/input.iso of=/dev/sdx oflag=sync status=progress` to write the ISO to the USB stick. Don't forget to replace the two paths with the correct ones.
+1. Insert the USB stick into the target computer and boot it from there
+
+As soon as you can see the Arch Linux prompt, you are ready for the next step.
+
+## Check for UEFI support
```

Comment:
It depends... The user needs to be able to run this command from somewhere. If the computer doesn't contain a running Linux installation already, this won't work. So, to keep it simpler, I would leave it here. Otherwise, it might be confusing.